### PR TITLE
[RBAC] Add default arg for organization foreign key fields

### PR DESCRIPTION
**Pull request by Dostonbek1**

### CODE REVIEW

**Comment by benthomasson** [comment]
@Dostonbek1 where is the default organization created?

**Review by benthomasson — APPROVED** [review]

**Review by mkanoor — APPROVED** [review]

File: src/aap_eda/services/activation/manager.py [hunk 1]
```diff
@@ -1053,6 +1053,7 @@ def _create_activation_instance(self):
             "name": self.db_instance.name,
             "status": ActivationStatus.STARTING,
             "git_hash": git_hash,
+            "organization": self.db_instance.organization,
```
Comment (AlanCoding) [inline src/aap_eda/services/activation/manager.py:1056]: Calling it out in a comment, this is a case where the child inherits the organization from its parent

File: src/aap_eda/wsapi/consumers.py [hunk 2]
```diff
@@ -234,6 +240,7 @@ def insert_audit_rule_data(self, message: ActionMessage) -> None:
                 fired_at=message.rule_run_at,
                 job_instance_id=job_instance_id,
                 status=message.status,
+                organization=activation_org,
```
Comment (AlanCoding) [inline src/aap_eda/wsapi/consumers.py:243]: second case `AuditRule` (2)

**Review by AlanCoding — CHANGES_REQUESTED** [review]
Looking for Rulebook to inherit org from project (3)

**Review by AlanCoding — APPROVED** [review]

**Review by benthomasson — APPROVED** [review]

**Pull request merged**